jarviscli/plugins/international_day.py

```diff
@@ -0,0 +1,128 @@
+    International Day of the current date or of the
+    input date the user enters.
+    """
+    def __call__(self, jarvis, s):
+        """
+        Calls the needed methods to execute the plugin.
+        """
+        jarvis.say("\nWhich date would you like to know"
+                   " there is an international day for? ")
+        # Formats the printed message
+        message = Fore.RESET + Fore.YELLOW + \
+            "[1]" + Fore.RESET + " For today's!\n"
+        message += Fore.RESET + Fore.YELLOW + "[2]" + Fore.RESET + \
+            " For another day...\n"
+        jarvis.say(message)
+        # Takes the user's input choice.
+        choice = jarvis.input("Please enter your choice: ")
+        # Loop to validate the input choice.
+        while True:
+            try:
+                # Find International Day for the current date.
+                if choice == "1":
+                    current_date = self.find_date()
+                    current_day, current_month = self.split_date(current_date)
+                    # Open the csv file that contains the International Days
+                    # which is located in the data folder.
+                    with open(r'jarviscli\data\international_days.csv')\
+                            as csv_file:
+                        print_message = \
+                            self.find_international_day(current_date,
+                                                        current_day,
+                                                        current_month,
+                                                        csv_file)
+                        print(print_message)
+                    break
+                # Find International Day for the given date.
+                elif choice == "2":
+                    # Takes the user's input date.
+                    another_date = jarvis.input("\nPlease enter the date"
+                                                " you like"
+                                                " (ex. 2021-09-21): ")
+                    # Validates the input date to be in the proper format.
+                    while not self.validate_input_date(another_date):
+                        jarvis.say("Please enter a valid date! ",
+                                   Fore.YELLOW)
+                        another_date = jarvis.input()
+                    # Spits the input date into month and day variables.
+                    # ex. 2021-09-21 --> month = 09, day = 21
+                    month = another_date.split('-')[1]
+                    day = another_date.split('-')[2]
+                    with open(r'jarviscli\data\international_days.csv')\
+                            as csv_file:
+                        print_message = self.find_international_day(
+                            another_date, int(day), int(month), csv_file)
+                        print(print_message)
+                    break
+                else:
+                    raise ValueError
```

Comment:
Hmm, actually Exceptions should only be thrown rarely and only to notify code up the call-stack, that something unexpected happened, that cannot be fixed in the sub-routine.

Did actually confuse me:

```
Some error occurred, please open an issue on github!
Here is error:

Traceback (most recent call last):
  File "/home/phil/git/Jarvis/jarviscli/plugins/international_day.py", line 49, in __call__
    self.find_international_day(current_date,
  File "/home/phil/git/Jarvis/jarviscli/plugins/international_day.py", line 103, in find_international_day
    for row in csv_reader:
  File "/home/phil/git/Jarvis/env/lib64/python3.8/codecs.py", line 322, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x92 in position 972: invalid start byte
```

There is something strange with the encoding of your csv-file (apparently some strange characters; might be linux <-> windows error?). Maybe try to check the file is really utf-8 encoded (but don't worry, if not I'll fix this for you).

The really interessting stuff; because of the above Exception I got this:

```
~> Hi, what can I do for you?
international day

Which date would you like to know there is an international day for? 
[1] For today's!
[2] For another day...

Please enter your choice: 1
Oops! That was no valid choice. Try again...
```

Which of course is not true; my choice was valid ;).